std/math.d

```diff
@@ -2178,9 +2178,10 @@ creal expi(real y) @trusted pure nothrow @nogc
  *      $(TR $(TD $(PLUSMN)$(NAN)) $(TD $(PLUSMN)$(NAN)) $(TD int.min))
  *      )
  */
-T frexp(T)(T value, out int exp) @trusted pure nothrow @nogc
+T frexp(T)(T _value, out int exp) @trusted pure nothrow @nogc
```

Comment:
Would it be ok to not use names start with underscores as function parameters? This comes up in the public documentation.